main/autobot/core/core.js

```diff
@@ -1,53 +1,34 @@
-var tracker = require('../project_trackers/jira_tracker');
+var jiraResource = require('../resources/jira');
+var access = require('../../lib/resource_accessor').access;
```

Comment:
With es7 this could become something like

`import {access} from resource_accessor`